File: DAY-1/solution.py
num=int(input("enter the number:"))
orgnum=num
rev=0
revnum=0;
while(num!=0):
    rev=num%10
    revnum=revnum*10+rev
    num=num/10
if(orgnum==revnum):
    print("the num is palindrome")
else:
    print("the num is not palindrome")

File: DAY-1/solution.c
#include<stdio.h>
int main()
{
    int num,orgnum,rem,revnum=0;
    printf("Enter a number:");
    scanf("%d",&num);
    orgnum=num;
    while(num!=0)
    {
        rem=num%10;
        revnum=revnum*10+rem;
        num /= 10;
    }
    if(orgnum==revnum){
        printf("%d is a palindrome",orgnum);
    }
    else{
        printf("%d is not a palindrome",orgnum);
    }
}